### ISSUE TITLE
"twenty nineteen" is converted to 39

### ISSUE THREAD

**Issue by tijoseymathew** [ShailChoksi/text2digits#12]
MWE of bug
```python
from text2digits import text2digits
t2d = text2digits.Text2Digits()
t2d.convert("twenty nineteen")
#'39'
```
Can be solved by adding `and result < 10` to
[https://github.com/careless25/text2digits/blob/1da9afde47257bb5c7a0530f022719798a570d41/text2digits/text2digits.py#L178](code)
```python
                   if is_tens and (word not in UNITS or word == "ten") and (word not in SCALES) and result < 10:
```

**Comment by vman049** [comment]
Thanks @tijoseymathew for pointing out the bug and @careless25 for writing this package. I'm still getting the error after adding the code above. Are you sure that is the correct fix or does something else need to be done?

**Comment by safa0** [comment]
The code suggestion does not work, but if you fetch the latest version or fetch the pull request: 
https://github.com/careless25/text2digits/pull/17
it will address this bug.

**Comment by safa0** [comment]
This is fixed in 0.0.8
